build.psm1

```diff
@@ -407,12 +405,18 @@ Fix steps:
         return
     }
 
+    # Set $RuntimeRID as Win7-x64 or Win7-x86 so the package works on all supported Windows OS versions.
+    switch ($Runtime) {
+        'win-x64' { $RuntimeRID = 'win7-x64' }
+        'win-x86' { $RuntimeRID = 'win7-x86' }
+    }
+
     # set output options
     $OptionsArguments = @{
         CrossGen=$CrossGen
         Output=$Output
         FullCLR=$FullCLR
-        Runtime=$Runtime
+        Runtime=$RuntimeRID
```

Comment:
I think this is consistent with dotnet's portable runtime concept.  unfortunately, they have defined the portable runtimeid for windows to be `win7-x86` and `win7-x64`